src/nunit.analyzers.tests/Extensions/ITypeSymbolExtensionsTests.cs

```diff
@@ -208,9 +208,12 @@ private static async Task<ImmutableArray<ITypeSymbol>> GetTypeSymbolAsync(string
 
             foreach (var typeName in typeNames)
             {
-                types.Add(rootAndModel.Model.GetDeclaredSymbol(rootAndModel.Node
-                    .DescendantNodes().OfType<TypeDeclarationSyntax>()
-                    .Where(_ => _.Identifier.ValueText == typeName).Single()));
+                INamedTypeSymbol? item = rootAndModel.Model.GetDeclaredSymbol(
+                    rootAndModel.Node
+                                .DescendantNodes().OfType<TypeDeclarationSyntax>()
+                                .Where(_ => _.Identifier.ValueText == typeName).Single());
```

Comment:
I was wondering if we should move this into a variable for readability. So something like

```
TypeDeclarationSyntax typeDeclaration = rootAndModel.Node
    .DescendantNodes()
    .OfType<TypeDeclarationSyntax>()
    .Where(_ => _.Identifier.ValueText == typeName)
    .Single();
INamedTypeSymbol? item = rootAndModel.Model.GetDeclaredSymbol(typeDeclaration);
...
```

Reply:
This code was more or less in there from the beginning, but you are right; splitting is make it more readable.
Done.